src/main/java/io/cryostat/core/net/Credentials.java

```diff
@@ -39,26 +39,26 @@
  * SOFTWARE.
  * #L%
  */
-package com.redhat.rhjmc.containerjfr.core.net;
+package io.cryostat.core.net;
 
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 
 public class Credentials {
 
-    private final transient String username;
-    private final transient String password;
+    private final String username;
+    private final String password;
 
     public Credentials(String username, String password) {
         this.username = username;
         this.password = password;
     }
 
-    String getUsername() {
+    public String getUsername() {
         return username;
     }
 
-    String getPassword() {
+    public String getPassword() {
         return password;
     }
```

Comment:
Is this visibility change intentional?

Reply:
No, that looks like a mistake - not sure how that snuck in. Thanks for catching it, very sharp eye!

Reply:
(that change is used for something else in a testing branch related to autorules, but I don't know how it ended up in this patch)